MLPerf_Results_Messaging_Guidelines.md

```diff
@@ -0,0 +1,194 @@
+# MLPerf™ Results Messaging Guidelines
+
+The MLPerf™ name and logo are trademarks of MLCommons™ Association (“MLCommons”) in the United States and other countries. The MLPerf name and logo are used to identify the source of MLCommons Association benchmarks. You may not use the MLPerf name or logo, except in a factual and non-trademark manner solely to accurately represent your benchmark score, in accordance with the following usage terms. Nothing in these MLPerf Results Messaging Guidelines (the “Guidelines”) grants you a license to use the MLPerf trademark or logo, or any other right or interest in or to the MLPerf trademark or logo. 
+
+# I.	Use of MLPerf™ Benchmark for Non-MLCommons Reviewed Results
```

Comment:
Manual numbering makes the document hard to edit. Can we use auto-numbering like the policies document?

Reply:
Yes, that's a great idea. I totally forgot about that. Let me see if I can figure out how to set that up. Can I just emulate https://github.com/mlcommons/policies/blob/master/submission_rules.adoc ?

Reply:
I think you'll need to switch from .md to .adoc.

Reply:
(... and then add the same directives as submission_rules.adoc.)